### ISSUE TITLE
Improve labels on `textarea` inputs within VRA drawers

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-VraCore#112]
Currently, `textarea` inputs within drawers use `aria-label="Content"` for their accessible names. It would be a vast improvement to screen reader announcements to instead describe these fields more specifically according to their corresponding onscreen labels (like in the example below, "Author").

<img width="1264" height="553" alt="Admin dashboard. An open drawer on the VRA Core tab of an item edit page. Browser dev tools are open, with hover over the HTML for a textarea input within a drawer labelled 'Author,' which is in view" src="https://github.com/user-attachments/assets/0924f2a6-3b49-47db-8c59-a725bdd8dc4b" />

**Comment by kimisgold** [comment]
These textareas sit within groups labeled by the property headings, which I think we've been okay with in other plugins. [Here's a related discussion for Contribution.](https://github.com/omeka/plugin-Contribution/issues/78#issuecomment-3276253603) I'm open to the possibility that this needs more detailed labeling than that, but want to make sure I understand why.

**Comment by aviannamiller** [comment]
Absolutely. I'm thinking it's a matter of that `aria-label` overriding some of those group relation announcements that you would otherwise get in a similar implementation. When I test with NVDA, JAWS, TalkBack, and VoiceOver on mobile, I am only encountering an announcement akin to "edit text, Content"; this is the case whether I tab forwards or backwards through the drawers. The same is true for forward tab navigation with VoiceOver on desktop, but navigating backwards (shift+tab) does then reveal that grouping (as "Content, edit text, Notes, group").

From there, though, would a direct link still need to be established between drawer name, as label, and textarea?

**Comment by kimisgold** [comment]
So I found that the `role="group"` and associated label were placed on the wrong container for one of the commonly used drawer markup blocks. They've been moved to the appropriate container. Can you check to see if that addresses what you're seeing?

**Comment by aviannamiller** [comment]
Structurally, things look good in the accessibility tree, but I am not experiencing anything different with screen reader announcements (across all tested screen readers). It does appear that the `aria-label` is the only component recognized within/as the accessible name for each textarea.

Just for the sake of comparison, the inputs (as well as the remove/undo buttons) on Contribution's drawers do reference their associated drawer names in their announcements, though those elements look to be nested under those `.drawer-name` elements. I'm not sure how much of a difference that makes, though?

**Comment by zerocrates** [comment]
This textarea is inside a fieldset with no legend, which is an implicit group and is "closer" than the one for the outer container. Is that what's causing this perhaps?

**Comment by aviannamiller** [comment]
That sounds plausible. Also just not a good practice at all to have a fieldset without a legend, unless there's a direct (and appropriately applied) replacement for a label in there, which I don't think there is. If similar uses are identified anywhere else, we should be noting those to make changes accordingly.
